internal/notification/handlers/notification_worker.go

```diff
@@ -42,12 +43,54 @@ type NotificationWorker struct {
 	backOff  func(current time.Duration) time.Duration
 }
 
+func (w *NotificationWorker) NextRetry(job *river.Job[*notification.Request]) time.Time {
+	return w.now().Add(job.CreatedAt.Sub(w.now()))
```

Comment:
This doesn't look right, I think we need to use some version of the existing backoff policy.  
```suggestion
	return w.now().Add(w.backOff(job.AttemptedAt.Sub(job.CreatedAt)))
```
This isn't quite right either since the existing `backoff` fn expects the last delay, not the total delay. We might need to refactor the backoff policy to follow River's pattern which uses the attempt count (based on `len(job.Errors) + 1`) to determine the next delay.

Reply:
does it even need to be that configurable or could we even go with the default backoff: https://riverqueue.com/docs/job-retries#client-retry-policy

Reply:
the default would also work for me

Reply:
Yep makes sense. I don't think their default is the best for all use cases, but we can definitely revisit later.